pkg/daemon/ceph/client/image.go

```diff
@@ -73,6 +75,11 @@ func CreateImage(context *clusterd.Context, clusterName, name, poolName string,
 	imageSpec := getImageSpec(name, poolName)
 
 	args := []string{"create", imageSpec, "--size", strconv.Itoa(sizeMB)}
+
+	if dataPoolName != "" {
```

Comment:
what would the example look like? I'm assuming:
- Create a replica pool CRD
- Create an EC pool CRD
- The storage class will specify both `pool` and `dataPool`.

Is dataPool required to be an EC pool? Or could it be a different replicated pool from the metadata pool? If it must be an EC pool, maybe we should name it as such.

Reply:
Yes, exactly that flow.
The `replica` pool would be the metadata pool and the EC pool the `dataPool`.

As far as I understand one can generally use two different pools even for `replicated` pools, e.g. `rbd create replicatedpool1/myimage --data-pool=replicatedpool2 [OPTIONS]`. It doesn't seem to be a EC "only" thing.